go-controller/pkg/ovn/policy.go

```diff
@@ -95,17 +97,22 @@ func (oc *Controller) syncNetworkPolicies(networkPolicies []interface{}) {
 	}
 }
 
-func addAllowACLFromNode(logicalSwitch string, mgmtPortIP net.IP) error {
+func addAllowACLFromNode(logicalSwitch string, mgmtPortIP net.IP, ovnNBClient goovn.Client) error {
 	ipFamily := "ip4"
 	if utilnet.IsIPv6(mgmtPortIP) {
 		ipFamily = "ip6"
 	}
 	match := fmt.Sprintf("%s.src==%s", ipFamily, mgmtPortIP.String())
-	_, stderr, err := util.RunOVNNbctl("--may-exist", "acl-add", logicalSwitch,
-		"to-lport", defaultAllowPriority, match, "allow-related")
-	if err != nil {
-		return fmt.Errorf("failed to create the node acl for "+
-			"logical_switch=%s, stderr: %q (%v)", logicalSwitch, stderr, err)
+
+	priority, _ := strconv.Atoi(defaultAllowPriority)
+	aclcmd, err := ovnNBClient.ACLAdd(logicalSwitch, "to-lport", match, "allow-related", priority, nil, false, "", "")
+	if err == nil {
+		// NOTE: goovn.ErrorExist is returned if the ACL already exists, skip returning an error
+		if err = ovnNBClient.Execute(aclcmd); err != nil && err != goovn.ErrorExist {
+			return fmt.Errorf("failed to create the node acl for logical_switch: %s, %v", logicalSwitch, err)
+		}
+	} else {
+		return fmt.Errorf(err.Error())
```

Comment:
might help to give more context to the generic command error by say logging similar to L112.

Reply:
Good suggestion, will add the context